files/en-us/web/api/window/showopenfilepicker/index.html

```diff
@@ -2,11 +2,11 @@
 title: Window.showOpenFilePicker()
 slug: Web/API/Window/showOpenFilePicker
 tags:
-- File
-- File System Access API
-- Method
-- Window
-- working with files
+  - File
+  - File System Access API
+  - Method
+  - Window
+  - working with files
 ---
 <div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("File System Access API")}}
```

Comment:
```suggestion
<div>{{securecontext_header}}{{DefaultAPISidebar("File System Access API")}}
```

Maybe we can remove Draft here? This page looks reasonably complete.

Reply:
> Maybe we can remove Draft here? This page looks reasonably complete.

Yup — so removed